src/main/kotlin/gg/essential/vigilance/Vigilance.kt

```diff
@@ -1,16 +1,8 @@
 package gg.essential.vigilance
 
-import gg.essential.elementa.effects.StencilEffect
-
 object Vigilance {
-    private var initialized = false
-
     @JvmStatic
     fun initialize() {
```

Comment:
Perhaps this should be annotated with deprecation level hidden to prevent from people adding this in the future.